packages/subgraph/src/osx/pluginSetupProcessor.ts

```diff
@@ -56,44 +49,26 @@ export function handleInstallationPrepared(event: InstallationPrepared): void {
   }
 
   //////////////////////////////////////////////////////////////
-  // Index preparation
+  // Index plugin
```

Comment:
Can you remind me why the `PluginSetupProcessor` event handling code (+tests) should sit in the template and not, lets say, osx-commons?

Reply:
yes, moving this code to commons seems like a nice idea,
but then we need to create entities types on the common schema, and template subgraph looses flexibility
and the types will be different between what commons returns and what template expects.

@mathewmeconry WDYT?